docs/help-topics/rhtap-secrets/rhtap-secrets.yml

```diff
@@ -0,0 +1,44 @@
+# Name is an internal name. Title displays in the UI as a side panel title.
+# Leave tags empty for now. Tags will specify where in the app descriptions will be available.
+# Links to be external only. We don't know yet whether referencing other side panels will be supported but referencing to in-depth docs is expected to be supported.
+# Note that this topic is for LEARN MORE.
+
+- name: rhtap-secrets-secrets
+  tags:
+  title: Secrets
+  content: |-
+    Add and manage your build and deployment secrets and their related configurations. You can add a secret at the workspace level to keep your data private. All secrets are stored using AWS Secrets Manager.  
```

Comment:
Can you add a header or something else to ease into this sentence? It feels like we jump right into things and I need more context. For example, even something like "You can use secrets in {ProductName} to do X, Y, and Z. To add and manage your secrets..."